### ISSUE TITLE
Remove source from extended properties observed data

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/PK-Sim#2167]
This does not bring anything and exposes some potential user identity

    "Name": "Source",
          "Value": "C:\\Users\\xxxx\\Documents\\yyy\\Data-collection.xlsx.Normal data"

where xxx and yyy are in this case username

I would vote for removing it from the import process

**Comment by msevestre** [comment]
Thoughts @Yuri05 @sfrechen @StephanSchaller @JanSchlender

**Comment by msevestre** [comment]
The same with the source property for each column. This is now redundant with the ImporterConfiguration and is not visible anyways

**Comment by Yuri05** [comment]
I habe another proposal:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1559

Completely removing the **Source** from the Meta data might be problematic. E.g. the source will be required for the implementation of ["observed data as reference"(https://github.com/Open-Systems-Pharmacology/Suite/issues/46)

**Comment by msevestre** [comment]
I don't quite understand your suggestion. You want to keep the `FileName` instead of the file Path? It's already a meta data called FILE

At any rate, we are not anywhere near implementing reference stuff so we should remove it now (because it's really garbage) and add it back if we need it. Observed Data saved NOW won't be compatible with referenced data anyways

**Comment by msevestre** [comment]
@Yuri05 Are you ok with us removing this nonsense for now and add it back if we need when we are ready? For now, this is IMO not ok

**Comment by Yuri05** [comment]
ok, let's remove for now.
@sfrechen @JanSchlender FYI

**Comment by StephanSchaller** [comment]
@PavelBal , I guess this was also an issue for a recent project added to the qualification framework. So I would agree to remove these for now.

**Comment by PavelBal** [comment]
Also for removing.

**Comment by msevestre** [comment]
done

**Comment by PavelBal** [comment]
This is only for newly added data, or? At least in MoBi, ald data sets have the "Source" metadata.

**Comment by msevestre** [comment]
yes. Meta data created won't have it anymore But since you can add meta data by hand, we cannot remove anything
